defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+satellite_admin_username: admin
+satellite_admin_password: CHANGEME
```

Comment:
I don't believe that the app will fail. I *think* it will auto-generate a password and leave it behind in the config file. But I have writing to the config file prevented, so the password would never be found.

I can add an assertion to the role to make sure this value is defined, and leave it commented out here. Would that work?

Reply:
I like that plan.